autoload/go/cmd.vim

```diff
@@ -1,6 +1,17 @@
 function! go#cmd#autowrite() abort
   if &autowrite == 1 || &autowriteall == 1
     silent! wall
+  else
+    for l:nr in range(0, bufnr('$'))
+      if buflisted(l:nr) && getbufvar(l:nr, '&modified')
+        " Sleep one second to make sure people see the message. Otherwise it is
+        " often immediacy overwritten by the async messages (which also don't
+        " invoke the "hit ENTER" prompt).
+        call go#util#EchoWarning('[No write since last change]')
+        sleep 1
```

Comment:
Will sleep block any other running async calls? Seems like a dangerous thing to add in that case.